### PR TITLE
Potential Solution for WP Multisites with Integrated Composer Sub Site Redirects

### PR DESCRIPTION
**Pull request by whitneymeredith**
Closes # [DOC-383](https://getpantheon.atlassian.net/jira/software/projects/DOC/boards/186?selectedIssue=DOC-383)

## Summary

**[Integrated Composer](https://pantheon.io/docs/guides/integrated-composer#troubleshooting-code-syncs-and-upstream-updates)** - Created a partial outlining the potential solution and added to the [Integrated Composer Troubleshooting ](https://pantheon.io/docs/guides/integrated-composer#troubleshooting-code-syncs-and-upstream-updates) section and the [WordPress Site Networks Troubleshoot](https://pantheon.io/docs/guides/multisite/debug/) page.

## Effect

The following changes are already committed:

*
*

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] List any outstanding work here

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-7188](https://pr-7188--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `50` | `63` | `13` |
| Best Practice Score | `79` | `79` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-7188` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/34840/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/34840/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-7188--pantheon-docs.my.pantheonfrontend.website/docs)

**Comment by whitneymeredith** [comment]
[Preview for Integrated Composer](https://pr-7188--pantheon-docs.my.pantheonfrontend.website/docs/guides/integrated-composer#troubleshooting-code-syncs-upstream-updates-and-redirect-errors) and [Preview for WordPress Site Networks](https://pr-7188--pantheon-docs.my.pantheonfrontend.website/docs/guides/multisite/debug/#potential-wordpress-multisite-with-integrated-composer-subdirectory-configuration-solution)

File: source/partials/configure-wp-site-networks-with-integrated-composer.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+We support WordPress multisites with subdirectories. However, if you are working with a composer-based Multisite ​setup, your sub sites might produce redirect errors. Your reference URLs might be incorrect if your WordPress core file path looks like this: `/code/web/wp`.
+
+<Alert title="Note"  type="info" >
+
+The solution outlined below has worked for select customers, and may not work for your specific configuration / environment.
+
+</Alert>
+
+Move your WordPress core files from `/code/web/wp` to `web/` root by using the [example WordPress Composer script](https://github.com/pantheon-systems/example-wordpress-composer/blob/f73fe27153ba6a772e7b316edd3442ed1de7c29d/scripts/composer/cleanup-composer).
```
Comment (greg-1-anderson) [inline source/partials/configure-wp-site-networks-with-integrated-composer.md:11]: The documentation should be consistent about the use of `/code`, i.e. `/code/web/wp` is used in the first path here, and in the previous example, so `web/` should be `/code/web` for consistency (or remove `/code/` from all of the paths where it appears).

**Comment by wordsmither** [comment]
lgtm

**Pull request merged**